doc/developers/contributing.rst

```diff
@@ -520,6 +522,32 @@ A good etiquette to take over is:
   new PR to the old one. The new PR should be created by pulling from the
   old one.
 
+Stalled and Unclaimed Issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Issues may have conversation that has not yet resulted in a pull request,
+or that is linked to a :ref:`stalled pull request <stalled_pull_request>`.
+In general, if a pull request has already been opened, it is best to follow
+the procedure described in the :ref:`stalled_pull_request` section rather
+than working in an issue.
+
+If, however, a pull request has not yet been opened, we also provide a
+suggested workflow for claiming and working on stalled or unclaimed issues:
+
+* First, to **determine if an issue is claimed**:
+
+  * Check for linked pull requests
+  * Check the conversation to see if anyone has said that they're working on
+    creating a pull request
+
+* If a contributor comments on an issue to say they are working on it,
+  a pull request is expected within 2 weeks (new contributor) or 4 weeks
+  (contributor or core dev), unless an larger time frame is explicitly given.
+  Beyond that time, another contributor can take the issue and make a
+  pull request for it. We encourage contributors to comment directly on the
+  stalled or unclaimed issue to let community members know that they will be
+  working on it.
+
```

Comment:
maybe add:

* If a PR exists for the issue but it is "stalled" according the the guidelines explained in the previous section, the contributor can take the issue and continue that PR.

Reply:
Thanks for the suggestions ! I've moved down the previous note into this section.
Please let me know if you have any feedback on this revision !